game.rb

```diff
@@ -0,0 +1,50 @@
+require './environment.rb'
+
+class Game
+    include Environment
+    attr_accessor :board, :depot, :control, :dictionary
+    def initialize
+        @board = {
+            :players =>{},
+            :control =>{},
+            :status => nil
+        }
```

Comment:
This kind of hashes may be stated like `@board = { players: {}, control: {}, status: nil }`